cypress/e2e/pages.cy.ts

```
@@ -0,0 +1,57 @@
+    cy.get('img[src="/screencaptures/preview-pwn2.gif"]').should('exist')
+
+    // Test Chatbot Builder previews
+    cy.visit('/preview/chatbot1')
+    cy.get('.preview').should('be.visible')
+    cy.get('img[src="/screencaptures/preview-chatbot1.gif"]').should('exist')
+    
+    cy.visit('/preview/chatbot2')
+    cy.get('.preview').should('be.visible')
+    cy.get('img[src="/screencaptures/preview-chatbot2.gif"]').should('exist')
+  })
+}) 
```

ESLint

[error] Newline required at end of file but not found.

(@stylistic/eol-last)